client/webserver/site/src/js/orderutil.ts

```diff
@@ -135,41 +137,41 @@ export function matchStatusString (m: Match) {
     // When revoked, match status is less important than pending action if still
     // active, or the outcome if inactive.
     if (m.active) {
-      if (m.redeem) return 'Revoked - Redemption Sent' // must require confirmation if active
+      if (m.redeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REDEMPTION_SENT }) // must require confirmation if active
```

Comment:
Seems like you'd need to `intl.prep` the `status` value too, no?

```Typescript
intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.prep(intl.ID_MATCH_STATUS_REDEMPTION_SENT) })
```

Reply:
yeah, thanks for the review.